telescope/app.py

```diff
@@ -228,8 +229,17 @@ async def heartbeat(request):
         if not missing_features
         else f"missing features {', '.join(missing_features)}"
     )
+
+    # Bugzilla
     bz_ping = await request.app["telescope.tracker"].ping()
     checks["bugzilla"] = "ok" if bz_ping else "Bugzilla ping failed"
+
+    # Big Query
+    try:
+        checks["bigquery"] = (await utils.fetch_bigquery("SELECT 'ok';"))[0]
```

Comment:
If bigquery is having an outage, this will also fail for us. Do we want this in our heartbeat because of that? Is there a way for us to check the exception to see if we received an auth error or a different error?